apps/frontend/src/network/users/UserProfile.tsx

```diff
@@ -85,6 +87,22 @@ const User: React.FC<{}> = () => {
         isOwnProfile && tab
           ? join(url, tab, EDIT_CONTACT_INFO_PATH)
           : undefined,
+      onImageSelect:
+        currentUser?.id === id && tab
+          ? (file: File) =>
```

Comment:
Can we extract the ugly bits of this into some sort of `fileToDataUrl` util please